uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -234,7 +234,7 @@ sub getSessionID {
 	# fixme ask Jon ... is this still useful?
     # determine the centerID and new visit number (which is now deprecated) if getPSC() failed.
 	if($centerID == 0) {
-            $query = "SELECT IFNULL(MAX(VisitNo), 0)+1 AS newVisitNo, CenterID FROM session WHERE CandID=".$dbh->quote($subjectIDref->{'CandID'})." GROUP BY CandID";
+            $query = "SELECT IFNULL(MAX(VisitNo), 0)+1 AS newVisitNo, CenterID FROM session WHERE CandID=".$dbh->quote($subjectIDref->{'CandID'})." GROUP BY CandID, newVisitNo, CenterID";
```

Comment:
I ran the query directly in the backend, so maybe running it from the script 'masks' the error?
I am testing the query directly form the backend because my project configuration is such that the visit has to be created to be able to upload, so this was the fastest way to tets this.